packages/grid_client/src/helpers/types.ts

```diff
@@ -89,3 +89,41 @@ export interface ZmachineData {
   /** The list of the GPUs */
   gpu: string[] | undefined;
 }
+
+interface VM {
+  version: number;
+  contractId: number;
```

Comment:
Upon reviewing the newly created file `types.ts`, I noticed that there is an interface for the `ZOS Machine` called `ZMachineData` that supports most of the `VM` types. However, it requires some additional types.

We can add the extra types as optional fields, like this:

```ts
interface ZMachineData {
  ... // Old fields
  deploymentName: string;
  projectName: string;
  wireguard: string;
}
```

Reply:
I extended VM from ZmachineData.